gammapy/maps/wcsnd.py

```diff
@@ -599,3 +601,40 @@ def smooth(self, radius, kernel='gauss', **kwargs):
         image = copy.copy(self)
         image.data = data
         return image
+
+    def make_cutout(self, position, width, mode="strict", copy=True):
+        """
+        Create a cutout of a WcsNDMap around a given position.
+
+        Parameters
+        ----------
+        position : `~astropy.coordinates.SkyCoord`
+            Center position of the cutout region.
+        width : tuple of `~astropy.coordinates.Angle`
+            Angular sizes of the region in (lon, lat). If only one value is passed,
+            a square region is extracted. For more options see also `~astropy.nddata.utils.Cutout2D`.
+        mode : {'trim', 'partial', 'strict'}
+            Mode option for Cutout2D, for details see `~astropy.nddata.utils.Cutout2D`.
+
+        copy : bool, optional
+               If False (default), then the cutout data will be a view into the original data  array. 
+               If True, then the cutout data will hold a copy of the original data array.
+        
+        Returns
+        -------
+        cutout : `~gammapy.maps.WcsNDMap`
+            The cutout map itself
+        """
+
+        idx = (0,) * len(self.geom.axes)
+
+        cutout2d = Cutout2D(data=self.data[idx], wcs=self.geom.wcs,
+                            position=position, size=width, mode=mode, copy=copy)
```

Comment:
Note that the data passed to `Cutout2D` is not actually used, so it does not help make the copy here. Instead just add
```python 
if copy:
    data = data.copy()
``` 
to the end of the cutout method.